src/components/PackListTile.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import { Link, Icon, Surface } from '@newrelic/gatsby-theme-newrelic';
+import NewRelicIcon from '@newrelic/gatsby-theme-newrelic/src/icons/logo/newrelic.js';
+
+const PackListTile = ({
+  name,
+  description,
+  featuredImageUrl,
+  supportLevel,
+  to,
+  className,
+}) => (
+  <Surface
+    as={Link}
+    to={to}
+    className={className}
+    base={Surface.BASE.PRIMARY}
+    interactive
+    css={css`
+      display: grid;
+      border-radius: 0.25rem;
+      position: relative;
```

Comment:
I don't think we need this `position: relative` here, it can have some interesting interactions with Grid that I don't think we want https://blogs.igalia.com/mrego/2016/05/27/css-grid-layout-and-positioned-items/

Reply:
It won't let me comment on the grid version, but we should also remove it there.

Reply:
oooh ok cool i've removed those, and made a couple of CSS changes for dark mode in the newest commits